deployment/amazon-ec2/conf/defaults.yml

```diff
@@ -21,7 +21,7 @@ instance_type: m4.xlarge
 image: ami-05cf2265
 volume_type: standard
 key_name: metron-test #Change this
-ec2_host_tag: metron-test
+ec2_host_tag: metron-include-test
```

Comment:
So they're easy to see in the aws console. They get lost in the list
otherwise.

On Wed, Mar 9, 2016 at 7:40 AM, Nick Allen <notifications@github.com> wrote:

> In deployment/amazon-ec2/conf/defaults.yml
> <https://github.com/nickwallen/incubator-metron/pull/2#discussion_r55512151>
> :
>
> > @@ -21,7 +21,7 @@ instance_type: m4.xlarge
> >  image: ami-05cf2265
> >  volume_type: standard
> >  key_name: metron-test #Change this
> > -ec2_host_tag: metron-test
> > +ec2_host_tag: metron-include-test
>
> What is this new tag for? Each EC2 host already gets tagged with 'ec2'. Do
> we need another?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nickwallen/incubator-metron/pull/2/files#r55512151>.
>